src/components/atoms/BottomPopModal/BottomPopModal.jsx

```diff
@@ -0,0 +1,40 @@
+import { AiOutlineClose } from "react-icons/ai";
+import { useCallback, useEffect, useState } from "react";
+import { lockBodyScroll } from "./utils";
+
+const BottomPopModal = ({ onClose, children }) => {
+
+  const [opened, setOpened] = useState(false);
+
+  useEffect(() => lockBodyScroll(), []);
+  useEffect(() => {
+    setOpened(true);
+  }, []);
+
+  const onCloseModal = useCallback(() => {
```

Comment:
리액트에서는 prop 에는 on prefix 내부 함수에는 handle prefix 를 붙이는 컨벤션이 있습니다.
```suggestion
  const handleCloseModal = useCallback(() => {
```